tools/cabana/messageswidget.cc

```diff
@@ -106,7 +106,7 @@ QVariant MessageListModel::data(const QModelIndex &index, int role) const {
   if (role == Qt::DisplayRole) {
     switch (index.column()) {
       case 0: return msgName(id);
-      case 1: return id;
+      case 1: return id.toString(); // TODO: put source and address in separate columns
```

Comment:
Noticed the "bus:addr" format is confusing for new users. But functional changes are for a separate PR.